services/graphql-server/src/graphql/resolvers/email/newsletter.js

```diff
@@ -7,6 +8,10 @@ module.exports = {
    *
    */
   Mutation: {
+    /**
+      * Setting an key in attributes value to null,
+      * will subsequently remove that key on the respective Newsletter Product,
+    */
```

Comment:
Sorry if I was unclear -- the comment should be added to the GraphQL field definition. This is fine to keep, but fix the indentation and include a description of the mutation itself -- updates email newsletter provider fields, etc.

For the GraphQL doc, something [like this](https://github.com/parameter1/mindful-cms/blob/main/graphql/managed/src/definitions/email-block.js#L64), but specific to the mutation and input logic. The doc line should be added above the `attributes` field in the type definition.

Reply:
All good! I wasn't sure which "GraphQL definition" you were talking about so I thought it made sense to put it on the mutation resolver, I've added a description to the mutation resolver as well as to the fields on the input definition. Do let me know if anything needs to be changed, improved or added anywhere!